.github/workflows/build-packages.yml

```
@@ -515,21 +560,139 @@ jobs:
       with:
         working-directory: ${{ github.workspace }}/jans-cedarling/bindings/cedarling_python
         command: build
-        args: --release --locked -i python3.11
+        args: --release --locked -i python3.9 python3.10 python3.11 python3.12
+        target: ${{ matrix.target }}
+        manylinux: ${{ matrix.manylinux }}
+        before-script-linux: yum install -y protobuf-compiler || dnf install -y protobuf-compiler || true
         maturin-version: "1.9.4"
 
+    - name: Smoke-test wheels
+      if: '!matrix.skip_smoke_test'
```

actionlint (1.7.12)

[error] property "skip_smoke_test" is not defined in object type {manylinux: string; name: string; os: string; target: string}

(expression)